src/components/Form/ActionButton/Prompt.js

```diff
@@ -111,11 +81,7 @@ ${mailFooter}`);
           ))}
         </select>
       )}
-      <div
-        className={`mail-section ${
-          showHeaderAndFooter || fullEditMode ? 'mail-section-opened' : ''
-        }`}
-      >
+      <div className={`mail-section mail-section-opened`}>
```

Comment:
`mail-section` and `mail-section-opened` css classes can be merged has there is no point keeping the "opened" option.